modules/hitlnext/src/backend/repository.ts

```diff
@@ -464,4 +472,34 @@ export default class Repository {
       { count: conditions.limit, sortOrder: [{ column: 'id', desc: true }] }
     )
   }
+
+  deleteHandoff = async (id: string) => {
+    return this.bp.database.transaction(async trx => {
+      const handoff = await this.getHandoff(id)
+
+      // We want delete the events since it contains
+      // the user's conversation.
+      await trx(EVENT_TABLE_NAME)
+        .del()
+        .whereIn('threadId', [handoff.agentThreadId, handoff.userThreadId])
+
+      await trx(WEB_MESSAGE_TABLE_NAME)
```

Comment:
In general, we try to avoid dependencies between modules. Here there's a clear dependency between hitl and channel-web. when we do have a situation like this, we try to us local http api to interact between modules. I know this will make db query run out of the transaction but that's a choice we made

Reply:
+1. Add the delete conversation feature on the channel instead. This way, we could implement it in all channels with a similar route

Reply:
Right! I agree, what I did does work, but solely when using the web channel only. I removed the command and I am going to implement the deletion in this PR: #4288